CHANGELOG.md

```diff
@@ -124,7 +128,8 @@ Initial release on crates.io
 - Working basic flash downloader with nRF51.
 - Introduce cargo-flash which can automatically build & flash the target elf file.
 
-[Unreleased]: https://github.com/probe-rs/probe-rs/compare/v0.5.1...master
+[Unreleased]: https://github.com/probe-rs/probe-rs/compare/v0.6.0...master
+[0.5.1]: https://github.com/probe-rs/probe-rs/releases/tag/v0.6.0
```

Comment:
```suggestion
[0.6.0]: https://github.com/probe-rs/probe-rs/releases/tag/v0.6.0
```

Reply:
Good spot, thanks!

Reply:
Hmm, this still doesn't look resolved. Maybe something is wrong with the github updates.

Reply:
Hmm maybe. It definitely is now tho :) I just amended and repushed :)